src/TTCache.php

```diff
@@ -211,12 +211,14 @@ public function load(array $keys): LoadResult
 
         $loadedKeys = [];
         $validValuesResult = $this->cache->getMultiple($hkeys);
+        $tags = [];
         foreach ($validValuesResult->value() as $k => $tv) {
             $originalKey = $hashedKeysToOrigKeys[$k];
             $loadedKeys[$originalKey] = $keys[$originalKey];
             unset($keys[$originalKey]);
-            $this->rawTags(array_keys($tv->tags));
+            $tags = array_merge($tags, $tv->tags);
         }
+        $this->rawTags(array_keys($tags));
```

Comment:
It is generally not recommended to call `array_merge()` within a loop.

```suggestion
            $tags[] = $tv->tags;
        }
        $this->rawTags(array_keys(array_merge(...$tags)));
```